### DISCUSSION TITLE
Définition de bloc applicatif

### DISCUSSION THREAD

**Discussion by squarou** [dbarzin/mercator#910] [Q&A]
Bonjour,

J'ai une question concernant les blocs applicatifs. Pouvez-vous m'indiquer la définition qui se trouve derrière ? On la confond souvent avec la notion de processus métier et cela n'est pas clair pour nous.
Aux utilisateurs, comment utilisez-vous cette notion ?

Merci pour vos retour :)

**Comment by dbarzin** [comment]
L'ANSSI définit dans son guide sur la cartographie du système d'information un bloc applicatif comme étant "un ensemble d'applications". Cet ensemble peut être l'ensemble des applications bureautiques, RH, comptabilité, développement, administration, IT...
